server/src/main/java/io/spine/server/EventProducer.java

```diff
@@ -60,4 +62,9 @@
     default Nothing nothing() {
         return Nothing.getDefaultInstance();
     }
+
+    /**
+     * Obtains classes of the events produced by this object.
+     */
+    ImmutableSet<EventClass> producedEvents();
```

Comment:
Please move this declaration above `nothing()` to keep the important methods together and separate them from the convenience shortcut that is `nothing()`.